docs/tab-based-navigation.md

```diff
@@ -87,7 +87,7 @@ export default createBottomTabNavigator(
 Let's dissect this:
 
 * `tabBarIcon` is a property on `navigationOptions`, so we know we can use it on our screen components, but in this case chose to put it in the `createBottomTabNavigator` configuration in order to centralize the icon configuration for convenience.
-* `tabBarIcon` is a function that is given the `focused` state and `tintColor`. If you take a peek further down in the configuration you will see `tabBarOptions` and `activeTintColor` and `inactiveTintColor`. These default to the the iOS platform defaults, but you can change them here. The `tintColor` that is passed through to the `tabBarIcon` is either the active or inactive one, depending on the `focused` state (focused is active).
+* `tabBarIcon` is a function that is given the `focused` state and `tintColor`. If you take a peek further down in the configuration you will see `tabBarOptions` and `activeTintColor` and `inactiveTintColor`. These default to the the iOS platform defaults, but you can change them here. The `tintColor` that is passed through to the `tabBarIcon` is either the active or inactive one, depending on the `focused` state (focused is active). The orientation state `horizontal` is `true` when the device is in landscape, otherwise `false` for portrait.
```

Comment:
just a small nit: `tabBarIcon` is a function that is given the `focused` state and `tintColor` 

please change that to something like 

`tabBarIcon` is a function that is given the `focused` state, `tintColor` and `horizontal` param, which is a boolean.

for completeness. Thanks!

Reply:
Ah nice catch. Thank you.